kensu/utils/helpers.py

```diff
@@ -74,4 +74,9 @@ def flatten(d, parent_key='', sep='.'):
                         items.extend(flatten(i, new_key, sep=sep).items())
             else:
                 items.append((new_key, type(v).__name__))
-    return dict(items)
\ No newline at end of file
+    return dict(items)
+
+def logical_naming_batch(string):
```

Comment:
weird name. what does it do? add a comment?

if it replaces all digits by # (e.g. `/api/user-id/123` -> `/api/user-id/###`) we could use a simpler regexp replace for that.

Reply:
logical_naming_batch replace digits and group of digits with a simple #

Reply:
so `/api/user-id/123 -> /api/user-id/#` ?

Reply:
exact